frame/utility/src/benchmarking.rs

```diff
@@ -86,5 +112,33 @@ benchmarks! {
 		assert_last_event::<T>(Event::BatchCompleted.into())
 	}
 
+	submit_call_on_behalf_of {
+		let caller: T::AccountId = account("caller", SEED, SEED);
+		let call = Box::new(frame_system::Call::remark { remark: vec![] }.into());
```

Comment:
I think to get the maximum cost for the `Encode::encode` function we need to use a deeply nested call here.  
Or a rather long one, not sure what is more expensive.